Sem5Dz/Task_1.py

```diff
@@ -0,0 +1,9 @@
+#Напишите программу, которая на вход принимает два числа A и B, и возводит число А в целую степень B с помощью рекурсии.
+x = int(input("Введите возводимое число"))
+n = int(input("Введите степень возведения"))
+def power(x, n):
+    if n == 0:
+        return 1
+    else:
+        return power(x, n - 1) * x
```

Comment:
Блок кода `else` не обязателен, так как `return` завершает работу функции.
```suggestion
    return power(x, n - 1) * x
```